snappy/parts.go

```diff
@@ -35,9 +35,9 @@ import (
 // SystemConfig is a config map holding configs for multiple packages
 type SystemConfig map[string]interface{}
 
-// ServiceYamler implements snappy packages that offer services
-type ServiceYamler interface {
-	ServiceYamls() []ServiceYaml
+// AppYamler implements snappy packages that offer services
```

Comment:
I'm not sure if I'm more confused with or without the comment. :-)

s/snappy packages/snaps/
s/services/applications/

But I still don't know what it means. It's also somewhat surprising to see all of that part of the public interface. Perhaps that's all part of the fat that is going away with the single snap types you're working on?